packages/vapor/scss/tables/pagination.scss

```diff
@@ -34,23 +34,23 @@
 
     .change-page-link {
         padding: 0;
-        color: var(--medium-blue);
+        color: var(var(--deprecated-medium-blue));
```

Comment:
are these nested `var(var(...))` intentional?

Reply:
nope, bad copy-paste, glad you noticed, thanks jack!